Protocol.md

```diff
@@ -732,19 +750,34 @@ RSocket resumption exists only for specific cases. It is not intended to be an 
 1. Resumption is always initiated by the client and either allowed or denied by the server.
 1. Resumption makes no assumptions of application state for delivered frames with respect to atomicity, transactionality, etc. See above.
 
+### Resume Agreement
+
+If communication with ressumption support has been successfully initiated, the decision on whether a particular REQUEST should be resumed is done separatelly. For that purpose a Requester set (R)esume flag in the REQUEST(s) frame and if the Responder respond in the first FRAME with the (R)esume flag set, then this stream is assumed to be resumable. Otherwise, if any of the sides did not set flag (R)esume, the stream assumed to not be resumable. If a Requester has sent a REQUEST frame with the (R)esume flag set, but the connection is lost before the aggreement has made (first PAYLOAD received), such REQUEST frame has to be redelivered.
```

Comment:
```suggestion
If communication with ressumption support has been successfully initiated, the decision on whether a particular REQUEST should be resumed is done separately. For that purpose a Requester set (R)esume flag in the REQUEST(s) frame and if the Responder respond in the first FRAME with the (R)esume flag set, then this stream is assumed to be resumable. Otherwise, if any of the sides did not set flag (R)esume, the stream assumed to not be resumable. If a Requester has sent a REQUEST frame with the (R)esume flag set, but the connection is lost before the aggreement has made (first PAYLOAD received), such REQUEST frame has to be redelivered.
```